src/main/java/com/shopping_app/shoppingApp/model/Request/UserProfileUpdateRequest.java

```diff
@@ -0,0 +1,40 @@
+package com.shopping_app.shoppingApp.model.Request;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class UserProfileUpdateRequest {
+
+    private String name;
+    private String email;
+    private String phoneNumber;
+
+//    TODO getter and setter
```

Comment:
Can you address all the TODO's in this MR if possible?

Reply:
done